soc/espressif/common/Kconfig.wifi

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_FAMILY_ESPRESSIF_ESP32
+
+config ESP_WIFI_MAX_THREAD_PRIORITY
+	int "Maximum work queue thread priority"
```

Comment:
Make sure all descriptions have "Wi-Fi" in them (unless you feel it's redundant if you go for a menu) and spell it out consistently across descriptions/help strings.